demo/app.component.html

```diff
@@ -712,13 +798,23 @@ <h3 (click)="optsVisible = !optsVisible" style="cursor: pointer">
       <div *ngIf="chart.options.includes('curve')">
```

Comment:
The line interpolation options are visible for the equation plots demo. We should either hide them or use the options in the charts.